elements/drawtools/src/style.eox.js

```diff
@@ -15,3 +15,80 @@ button.polygon:before {
   content: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' fill='%23004170' viewBox='0 0 24 24'%3E%3Ctitle%3Eshape-polygon-plus%3C/title%3E%3Cpath d='M17,15.7V13H19V17L10,21L3,14L7,5H11V7H8.3L5.4,13.6L10.4,18.6L17,15.7M22,5V7H19V10H17V7H14V5H17V2H19V5H22Z' /%3E%3C/svg%3E");
 }
 `;
+
+export const styleListEOX = `
```

Comment:
I am wondering if there is a benefit to having this style as a separate export but within the same file. Maybe it would work to contain this style directly inside the `list.js`, but I don't have too strong of an opinion here.

Reply:
Done ✅